packages/datastore/datastore-codegen/tests/__snapshots__/Cli.test.ts.snap

```diff
@@ -0,0 +1,107 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`generate 1`] = `
+"export interface Note {
+    id: string;
+    title: string;
+    description: string;
+    comments: string[]
+    _version?: number;
+    _deleted?: boolean;
+}
+
+export type NoteCreate = Omit<Note, \\"id\\">;
+export type NoteChange =  Pick<Note, \\"id\\"> && Partial<NoteCreate>;
+
+export interface Comment {
+    id: string;
+    title: string;
+    description: string;
+    noteComment: string
+    _version?: number;
+    _deleted?: boolean;
+}
+
+export type CommentCreate = Omit<Comment, \\"id\\">;
+export type CommentChange =  Pick<Comment, \\"id\\"> && Partial<CommentCreate>;
+"
+`;
+
+exports[`generate 2`] = `
+"{
+  \\"Note\\": {
+    \\"name\\": \\"Note\\",
+    \\"version\\": 1,
+    \\"type\\": \\"object\\",
+    \\"primaryKey\\": \\"id\\",
+    \\"properties\\": {
+      \\"id\\": {
+        \\"type\\": \\"string\\",
+        \\"key\\": \\"id\\",
+        \\"isRequired\\": true,
+        \\"index\\": true,
+        \\"primary\\": true
+      },
+      \\"title\\": {
+        \\"type\\": \\"string\\",
+        \\"key\\": \\"title\\",
+        \\"isRequired\\": true
+      },
+      \\"description\\": {
+        \\"type\\": \\"string\\",
+        \\"key\\": \\"description\\",
+        \\"isRequired\\": true
+      },
+      \\"comments\\": {
+        \\"type\\": \\"string[]\\",
+        \\"key\\": \\"comments\\",
+        \\"isRequired\\": true
+      }
+    }
+  },
+  \\"Comment\\": {
+    \\"name\\": \\"Comment\\",
+    \\"version\\": 1,
+    \\"type\\": \\"object\\",
+    \\"primaryKey\\": \\"id\\",
+    \\"properties\\": {
+      \\"id\\": {
+        \\"type\\": \\"string\\",
+        \\"key\\": \\"id\\",
+        \\"isRequired\\": true,
+        \\"index\\": true,
+        \\"primary\\": true
+      },
+      \\"title\\": {
+        \\"type\\": \\"string\\",
+        \\"key\\": \\"title\\",
+        \\"isRequired\\": true
+      },
+      \\"description\\": {
+        \\"type\\": \\"string\\",
+        \\"key\\": \\"description\\",
+        \\"isRequired\\": true
+      },
+      \\"noteComment\\": {
+        \\"type\\": \\"string\\",
+        \\"key\\": \\"noteComment\\",
+        \\"isRequired\\": true
+      }
+    }
+  }
+}"
+`;
+
+exports[`generate 3`] = `
+"import { ModelJsonSchema } from \\"offix-datastore\\";
+import jsonSchema from \\"./schema.json\\";
+
+type Schema<T = any> = {
```

Comment:
Your source code is outdated. We have this in the core already.

Reply:
I'll rebase